docs/learn/learn-extrinsics.md

```diff
@@ -0,0 +1,152 @@
+nominate some validators. The signature of such extrinsic might lead to an event on the chain such
+as a reward payout to that nominator at the end of an era; this is an event inside the chain.
+
+## Extrinsics (a.k.a. Types of Transactions)
+
+Now that we introduced where the term _extrisic_ comes from, we can explain what extrinsics really
+are. In Substrate, a _"thing that could exist in the transaction pool"_ — more commonly known as "a
+transaction" in the non-Substrate world — can be one of 3 distinct types, all of which fall under a
+broader category called "extrinsics". Extrinsics is just a general term to mean "any information
+that originates from outside a runtime". These are:
+
+- **Signed transactions:** these must contain the signature of the account sending the inbound
+  request to the runtime. With signed transactions, the account used to submit the request typically
+  pays a transaction fee and must sign it using the account's private key.
+- **Unsigned transactions:** these don't carry any information about who submitted the transaction,
+  since the format of this type of transaction doesn't require a signature. You can define what
+  conditions must be met for such a transaction to be valid.
+- **Inherents:** are a special type of unsigned transaction made by block authors which carry
+  information required to build a block such as timestamps, storage proofs and uncle blocks.
+
+Here are some key differences between the different types of extrinsics:
+
+- Contrary to signed transactions, unsigned transaction types require implementing custom validation
+  logic which can consume more resources for checking validity compared to signed transactions.
+- Unsigned transactions have no economic deterrent to prevent spam or replay attacks, so custom
+  logic must account for protecting the network from these types of transactions being misused.
+- Inherents exist to address the need of adding some data to a block, whereas signed or unsigned
+  transactions exist to potentially change the state of the blockchain.
+
+## Metadata Updates
+
+:::warning Always check for Metadata Updates
+
+Before signing extrinsics with the Polkadot-JS Browser Extension and Parity Signer, always check for
+metadata updates. [This technical explainer video](https://youtu.be/gbvrHzr4EDY) will explain how to
+do it.
+
+:::
+
+Parity Signer updates the full metadata through the QR fountain while the extension updates the
+metadata index (the metadata is not loaded into it). As a consequence the process of updating
+metadata is different (you will notice that on the Signer app the update takes longer for example).
+Having outdated metadata on the Signer app will prevent you from signing while on the extension you
+will be able to click the sign button but the extrinsic will likely fail similarly of having an
+outdated Ledger app. In general, failing to update metadata will most likely result in you not being
+able to transfer funds.
```

Comment:
Don't limit it to "transferring funds" (there are a couple of instances like this). Instead: "not being able to sign extrinsics".